modules/ppcp-button/resources/js/modules/Helper/Utils.js

```diff
@@ -0,0 +1,84 @@
+export const toCamelCase = (str) => {
+    return str.replace(/([-_]\w)/g, function(match) {
+        return match[1].toUpperCase();
+    });
+}
+
+export const keysToCamelCase = (obj) => {
+    let output = {};
+    for (const key in obj) {
+        if (Object.prototype.hasOwnProperty.call(obj, key)) {
+            output[toCamelCase(key)] = obj[key];
+        }
+    }
+    return output;
+}
+
+export const strAddWord = (str, word, separator = ',') => {
+    let arr = str.split(separator);
+    if (!arr.includes(word)) {
+        arr.push(word);
+    }
+    return arr.join(separator);
+};
+
+export const strRemoveWord = (str, word, separator = ',') => {
+    let arr = str.split(separator);
+    let index = arr.indexOf(word);
+    if (index !== -1) {
+        arr.splice(index, 1);
+    }
+    return arr.join(separator);
+};
+
+export const debounce = (func, wait) => {
```

Comment:
There is also this version here https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/modules/ppcp-wc-gateway/resources/js/helper/debounce.js

dunno if we want to reuse it or copy it here (or vice versa). Using functions from other modules seems a bit weird and maybe should not be done for small utils.

Reply:
I had the same dilemma, maybe have them in lib. In this case I can remove the `debounce` function as it's not being used, was to have some coherence since I was adding `throttle`.

I will remove debounce so there is less confusion.